src/scripts/main.js

```diff
@@ -1,7 +1,29 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const body = document.querySelector('body');
+
+  const block = document.createElement('div');
+
+  block.setAttribute('class', `notification ${type}`);
+  block.style.top = posTop + 'px';
+  block.style.right = posRight + 'px';
+
+  const h2 = document.createElement('h2');
+
+  h2.setAttribute('class', 'title');
```

Comment:
Use `classList.add()` instead of `setAttribute()` for adding classes to the element.